### DISCUSSION TITLE
Problem with Chrome Security

### DISCUSSION THREAD

**Discussion by ciltocruz** [furybee/chrome-tab-modifier#377] [General]
Hello.

I bought a new computer and installed Chrome Tab Modifier.

It doesn't work.

The error I get is this:

```
Refused to load the script 'chrome-extension://e98cbfeb-3ded-4047-b873-61162833edcc/assets/content.js-D8gZ41FF.js' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

Is it my computer or is there a Chrome setting I need to change? I'm not sure where the problem is.
I don't know if it has anything to do with the new versions of the v3 manifest for chrome extensions

My Chrome version is:
Version 130.0.6699.3 (Official Build)

Thanks!

**Comment by sylouuu** [comment]
Hello @ciltocruz,

How do you know Tab Modifier is responsible of this error? Do you have the same of multiple websites?

Can you attach a screenshot please and what URL you visit?

Bests

**Comment by ciltocruz** [comment]
What I've done is put a title for the pages that contain "feedly.com". I've tried it in incognito with this extension only active and the error remains.

I'm attaching a couple of photos, although I'm not sure what to show.

The file that gives the error:
![image](https://github.com/user-attachments/assets/a2f1de38-c4fa-4220-a359-51be7a3b16c5)

The rule:
![image](https://github.com/user-attachments/assets/fa0f6339-f6f7-4f76-b21c-684c9a89bf7c)

**Comment by sebastienfontaine** [comment]
Hello,

I was unable to reproduce the issue. I have version 128.0.6613.138 (Official Build) (arm64). It’s strange, according to the Chrome releases, version 130 doesn’t exist : https://chromereleases.googleblog.com/

<img width="770" alt="image" src="https://github.com/user-attachments/assets/08abec60-45e5-4deb-8c46-7b9453514327">

Best regards,

**Reply by ciltocruz** [reply]
![image](https://github.com/user-attachments/assets/0c7fa4a8-d76a-4227-b084-f3af3d091008)

**Comment by sylouuu** [comment]
> I've tried it in incognito with this extension only active and the error remains.

Did you allow Tab Modifier to run in incognito mode? It's disabled by default like all extensions.

**Reply by ciltocruz** [reply]
> > I've tried it in incognito with this extension only active and the error remains.
> 
> Did you allow Tab Modifier to run in incognito mode? It's disabled by default like all extensions.

Yes, I enabled it manually.
